docs/reference/manifest/filename.md

```diff
@@ -0,0 +1,30 @@
+---
+title: FileName element in the manifest file
+description: ''
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# FileName element
+
+Specifies the file name of the equivalent XLL for custom functions in your web add-in.
+
+**Add-in type:** Custom function
+
+## Syntax
+
+```XML
+    <FileName>{filename}</FileName>  
+```
```

Comment:
Based on [this page](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/providername), this should look like:
```
<FileName>string</FileName>  
```